crates/ruff_linter/src/rules/pylint/rules/if_stmt_min_max.rs

```diff
@@ -29,6 +29,19 @@ use crate::fix::snippet::SourceCodeSnippet;
 /// highest_score = max(highest_score, score)
 /// ```
 ///
+/// ## Fix safety
+/// This fix is marked unsafe if it would delete any comments within the replacement range.
+///
+/// An example to illustrate where comments are preserved and where they are not:
+///
+/// ```pyi
+/// a, b = 0, 10
+///
+/// if a >= b: # deleted comment
+///     # deleted comment
+///     a = b # preserved comment
+/// ```
+///
```

Comment:
I don't think we generally include examples like this, but I don't really mind it if you want to keep it. The `preserved comment` here is technically after the replacement range, so it's a bit redundant with the explanation in your first line, but that's probably not obvious to most users.
```suggestion
/// An example to illustrate where comments are preserved and where they are not:
///
/// ```py
/// a, b = 0, 10
///
/// if a >= b: # deleted comment
///     # deleted comment
///     a = b # preserved comment
/// ```
///
```

If we do keep the example, I think this should just be `py`. I believe `pyi` is for stub files, where code like this would be pretty unusual, as far as I know.